### PR TITLE
DBZ-5757 Updates downstream steps to build KC custom container images

### PR DESCRIPTION
**Pull request by roldanbob**
[DBZ-5757](https://issues.redhat.com/browse/DBZ-5757)

Adds missing steps to examples for creating YAML to build custom Kafka Connect container images for each of the downstream connectors. This change has no bearing on the community version of the documentation. 

The change also replaces the callout legends that follow the Kafka Connect YAML file and KafkaConnect resource examples with tables. to workaround the customer portal rendering problem that results in multiple entries for each numbered item.

Please backport this change to 1.9

### CODE REVIEW

**Comment by jpechane** [comment]
@jcechace Please take a look

**Comment by roldanbob** [comment]
Also, @naros.

**Review by obabec — APPROVED** [review]
From my POV I think it looks alright, but I would wait for a review from Chris.

**Review by Naros — APPROVED** [review]

**Comment by jpechane** [comment]
@roldanbob Applied, thanks!

**Pull request merged**
